emperor/support_files/js/controller.js

```diff
@@ -253,10 +254,42 @@ define([
                                  scope.resize(scope.$divId.width(),
                                               scope.$divId.height());
                                }});
+
+    // Set up the context menu
+    this.$contextMenu = $.contextMenu({
+      selector: '.emperor-plot-wrapper',
```

Comment:
Unfortunately it pnly takes a jquery selector string, and not a full jquery element (https://swisnl.github.io/jQuery-contextMenu/docs.html). Is there any element ID that is unique to each instance?